spec/lib/action_account_spec.rb

```diff
@@ -30,7 +30,8 @@
         expect(stderr.string).to eq(<<~STR)
           *Task*: `review-pass`
           *UUIDs*: `#{user.uuid}`
-          *Messages*: * `#{user.uuid} : User is past the 30 day review eligibility.`
+          *Messages*:
+            * `#{user.uuid}`: User is past the 30 day review eligibility.
```

Comment:
my guess is that the leading space is not needed/wanted here

Reply:
Resolved in Slack conversation